lib/src/rules/unsafe_html.dart

```diff
@@ -134,12 +139,24 @@ class _Visitor extends SimpleAstVisitor<void> {
 
   @override
   void visitMethodInvocation(MethodInvocation node) {
-    var type = node.target?.staticType;
-    if (type == null) return;
-
     var methodName = node.methodName?.name;
     if (methodName == null) return;
 
+    // The static type of the target.
+    DartType type;
+    if (node.realTarget == null) {
+      // Implicit `this` target.
+      var enclosingElement = node.methodName.staticElement.enclosingElement;
```

Comment:
It looks like you need to be null-safe here:

https://travis-ci.org/github/dart-lang/linter/jobs/672562418

so, maybe:

`var enclosingElement = node.methodName.staticElement?.enclosingElement`

Reply:
yeah, fixed. Thanks!